blackbox/docs/sql/reference/grant.txt

```diff
@@ -4,25 +4,31 @@
 ``GRANT``
 =========
 
-Grants privilege to an existing user.
+Grants privilege to an existing user, on the whole cluster or a specific
+schema or table.
 
 Synopsis
 ========
 
 .. code-block:: psql
 
   GRANT { { DQL | DML | DDL [,...] } | ALL [ PRIVILEGES ] }
+  [{ ON SCHEMA schema_name |
+    ON TABLE  table_ident }]
   TO user_name [, ...];
 
 Description
 ===========
 
 ``GRANT`` is a management statement to grant a ``DQL``, ``DML`` or ``DDL``
-privilege on the cluster to one or many existing users.
+privilege on the cluster, or on a specific schema, or on a specific table
```

Comment:
pls rephrase to more generic e.g: is a management statement to grant a permission on a specific object to users.